RunningOrder/Shared/RunningOrderApp.swift

```diff
@@ -38,6 +39,38 @@ struct RunningOrderApp: App {
         dataPublisher: changesService.storyInformationChangesPublisher.eraseToAnyPublisher()
     )
 
+    @StateObject var appStateManager = AppStateManager()
+
+    let bugReportURL = URL(string: "https://github.com/worldline/RunningOrder/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D")!
```

Comment:
are you sure you want to force unwrap here when creating the URL ?

Reply:
yes, because the url is hard coded, and we know that the url is working. If it was dynamic I would agree with you to check it at runtime, but here we know the url is okay